test/e2e/cypress/e2e/host_details.cy.js

```diff
@@ -491,6 +491,43 @@ context('Host Details', () => {
       cy.wrap(user).as('user');
     });
 
+    describe('Check Execution', () => {
+      it('should forbid check execution when the correct user abilities are not present', () => {
+        cy.get('@user').then((user) => {
+          cy.createUserWithAbilities(user, []);
+          cy.login(user.username, password);
+        });
+        cy.visit(`/hosts/${selectedHost.agentId}/settings`);
+
+        cy.contains('button', 'Start Execution').should('be.disabled');
+
+        cy.contains('button', 'Start Execution').click({ force: true });
```

Comment:
`mouseover` ?

Reply:
It does not work because the button is disabled :/

Reply:
So maybe just better to check that the button is disabled?
We already test the tooltip in the unit tests

Reply:
the button is disabled because the checks are not selected, and wanda is not integrated and all the steps are missing, so I simply check that the tooltip it's not present, so it's disabled but not because of the policy, it's a little bit weird but rn it's what we have